ionic2/app/pages/about/about.page.scss

```diff
@@ -1,3 +1,7 @@
 .about {
 
 }
+
+ion-item a{
+  text-decoration: none;
```

Comment:
Be aware that this will overwrite all `ion-items`, not just the ones on the about page. Maybe you can add a CSS class to the `<ion-list>` in `about.page.html' and prefix your rules with that.